runtime/near-test-contracts/src/lib.rs

```diff
@@ -48,7 +48,11 @@ pub fn sized_contract(size: usize) -> Vec<u8> {
 /// on a specific protocol version, it should use [`base_rs_contract`].
 pub fn rs_contract() -> &'static [u8] {
     static CONTRACT: OnceCell<Vec<u8>> = OnceCell::new();
-    CONTRACT.get_or_init(|| read_contract("test_contract_rs.wasm")).as_slice()
+    let contract =
+        include_bytes!(concat!(env!("CARGO_MANIFEST_DIR"), "/res/", "test_contract_rs.wasm"))
+            .to_vec();
+
+    CONTRACT.get_or_init(|| contract).as_slice()
```

Comment:
I think you can remove the once cell for lazy initialization. `include_bytes!` already gives you a slice with static lifetime, so you should be able to just return it.
```rust
pub fn rs_contract() -> &'static [u8] {
    include_bytes!(concat!(env!("CARGO_MANIFEST_DIR"), "/res/", "test_contract_rs.wasm"))
}
```
This also avoids copying into heap memory. (in your code `to_vec` allocates new memory)

Reply:
Thanks for suggestion, applied.